api/db-migrations/000013_add_default_traffic_rule.down.sql

```diff
@@ -0,0 +1,50 @@
+WITH dangling_routes AS (
+    SELECT
+        id
+        , default_traffic_rule -> 'routes' AS routes
+    FROM
+        router_versions
+    WHERE default_traffic_rule -> 'routes' IS NOT NULL
+),
+cleaned_router_versions AS (
+    SELECT id, CASE WHEN traffic_rules = 'null' THEN '[]'::jsonb ELSE traffic_rules END AS traffic_rules
```

Comment:
Also here, I think we can skip the cleaning step. No harm having it as the dangling routes filter below will ignore these kinds of too but could simplify the query a little by leaving out this step.